### PR TITLE
feat: create user profile page

### LINKED ISSUE #329: feat: create user profile page
mobile:
image

desktop:
image

for exact details check [figma](https://www.figma.com/file/VtzXViXjXah3IuWn2xBRae/Website?node-id=736%3A51&t=1IvVfOuRpToT4y3i-0)

Check the already created components, and reuse them.

### PR DESCRIPTION
**Pull request by anish-patro**
Screenshot (141)
### Your checklist for this pull request

🚨Please review the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch** (right side). Don't request your default branch!
- [x] Make sure you are making a pull request against the **default branch** (left side). Also you should start _your branch_ off _default branch_.
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.
- [x] I have added necessary documentation (if appropriate)

### Linked Issues

Fixes #329 

### Description

Please describe your pull request.

❤️Thank you!

### Post merge checklist

- [ ] Follow steps from the [guidelines for contributing](https://github.com/Monday-Morning/project-tahiti/blob/main/CONTRIBUTING.md) to this repository.
- [ ] If you are a new contributor, ping in the thread and one of the maintainers will add you to all-contributors list.

### CODE REVIEW

**Comment by 120EE0692** [comment]
@anish-patro add images of your work in the description.

**Review by 120EE0692 — CHANGES_REQUESTED** [review]
image
@anish-patro add a [container](https://mui.com/material-ui/react-container/) component, around the user profile.

**Comment by anish-patro** [comment]
> @anish-patro add images of your work in the description

> @anish-patro add images of your work in the description.

File: client/src/screens/UserProfile.js [hunk 1]
```diff
@@ -150,17 +175,35 @@ const useStyles = makeStyles((theme) => ({
...
+    padding: '2px 6px',
+    cursor: 'pointer',
+    fontSize: '14px',
+    fontWeight: '400',
+    lineHeight: '20px',
+  },
   selectedOption: {
     color: `${theme.palette.secondary.main} !important`,
   },
+  selectedOptionM: {
```
Comment (sambit-sankalp) [inline client/src/screens/UserProfile.js:203]: Don't use random names to declare classes.

**Review by sambit-sankalp — CHANGES_REQUESTED** [review]

**Comment by rutajdash** [comment]
@sambit-sankalp @120EE0692 please proceed with this ASAP.

**Pull request closed without merging**